content/faq/startup-troubleshooting.md

```diff
@@ -0,0 +1,54 @@
+---
+title: LBRY Startup Troubleshooting
+category: troubleshooting
+order: 1
+---
+
+Internet connectivity and ports are key to running LBRY successfully since it is a Peer to Peer protocol that requires clients to see and speak to each other. Since individual local network configurations, firewalls, and other security settings vary greatly, LBRY may require some basic network/PC troubleshooting in order to function properly. Also since LBRY is in beta, there are certain configurations which we don't support yet but can offer workarounds (see below) or offer one on one [support](https://lbry.io/faq/how-to-report-bugs). 
+
+One of the common areas to check for start-up issues is the [LBRY Log file](https://lbry.io/faq/how-to-find-lbry-log-file). Except in extreme cases, a log file should be generated by the daemon startup process and will usually reveal clues on where to look. 
+
+## Connectivity and Ports
+LBRY operates on a couple different ports and if there are conflicts/firewall rules/security settings that prevent them from being utilized, the app/daemon will experience start-up issues. LBRY tries to employ port forwarding through the use of UPnP, which may be disabled on some routers. Either UPnP must be enabled or you have to manually forward the ports below (3333 TCP and 4444 UDP) in order to fully take advantage of the LBRY network. Only 1 PC on a network can have an open outside port, so if you are running multiple PCs with LBRY, they will need to be configured manually. 
+
+- Port 3333 - LBRY daemon runs and shares data on port 3333 (TCP) by default. Often times, this port can be already in use to due to mining software or other applications/services. 
+
+- Port 4444 - LBRY daemon utilizes port 4444 to stream and download data from the LBRY network. If this port is taken or not working properly, you generally won't have start-up issues, but instead, results in the inability to download any content. See the [streaming guide](https://lbry.io/faq/unable-to-stream) for the workaround. 
+
+- Port 50001 - LBRY utilizes an Electrum wallet clone which is lightweight wallet software that doesn't require the entire blockchain to operate. Instead, the wallet connects to a service which LBRY runs in order to improve the user experience and save on blockchain data storage. The connection happens over port 50001. LBRY may fail to start if this port is blocked by a firewall or network rules. 
```

Comment:
This is too much information. First two sentences can be trimmed to like 3 words - "LBRY wallet connections happen over port..."